provider/baked/src/export.rs

```diff
@@ -565,20 +565,20 @@ impl DataExporter for BakedExporter {
                     .any(|(_, ids)| ids.iter().any(|id| !id.locale.is_default()));
 
             let mut baked_values = deduplicated_values
-                .into_iter()
+                .iter()
                 .map(|(payload, ids)| {
                     stats.structs_count += 1;
                     stats.identifiers_count += ids.len();
                     stats.structs_total_size += payload.baked_size();
```

Comment:
issue: I don't think the size calculations are correct anymore

can be fixed in a follow-up

Reply:
Added a TODO